packages/core/email/package.json

```
@@ -34,7 +34,9 @@
     "lodash": "4.17.21",
     "prop-types": "^15.7.2",
     "react": "^17.0.2",
+    "react-dom": "^17.0.2",
     "react-intl": "6.2.8",
+    "react-router-dom": "5.3.4",
     "styled-components": "5.3.3",
```

```suggestion
    "react-intl": "6.2.8"
```

I think this list:

```json
    "react": "^17.0.2",
    "react-dom": "^17.0.2",
    "react-router-dom": "5.3.4",
    "styled-components": "5.3.3",
```

Should go in `peerDeps` and `devDeps` because we don't want more than one version being installed.

---

This comment is the same for all other packages, so i won't comment in their places but please check them :)